### ISSUE TITLE
iPhone X status bar does not have topBar color by default

### ISSUE THREAD

**Issue by RubenSandwich** [wix/react-native-navigation#3136]
### iPhone X status bar does not have navbar color by default

On the iPhone X, when presenting a navbar/topBar the status bar color should default to that of the navbar. However that currently does not happen, rather the status bar seems to default to the 

### Steps to Reproduce / Code Snippets / Screenshots

Here is what happens currently in React Native Navigation:
![simulator screen shot - iphone x - 2018-05-01 at 09 07 03](https://user-images.githubusercontent.com/1957315/39474166-94e7e3a2-4d20-11e8-9edb-9da03b276a4f.png)
(A screenshot taken from the simulator of the topBar, notice the white part above the navbar which is is where the iPhone X get its status bar background color from.)

---
### Environment
* React Native Navigation version: 2.0.2274
* React Native version: 0.52.2
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator, iOS 11.3, Both

**Comment by yogevbd** [comment]
@RubenSandwich  Can you post here your options code?

**Comment by RubenSandwich** [comment]
@yogevbd 

Passed in options:
```js
    topBar: {
      title: {
        color: '#FFFFFF',
        text: 'eventsScreenTitle',
      },
      background: {
        color: '#313E61',
      },
      leftButtons: [
        {
          id: 'EventsScreenNavBarButtons.Settings',
          disableIconTint: true,
          buttonColor: '#FFFFFF',
          icon: require('@images/gear.png'),
        },
      ],
    },
```

Setting the navigation root:
```js
Navigation.setRoot({
    stack: {
      children: [
        {
          component: {
            name: EVENTSSCREENID,
          },
        },
      ],
    },
  });
```

**Comment by gabrielvcbessa** [comment]
Same issue here, can I change the color manually?

**Comment by RubenSandwich** [comment]
Just an update on this, it seems to be fixed in v2.0.2348.

**Comment by Drewdizzy** [comment]
This is still an issue and no ones on the web explains how to get it right for iphone x simulator.

Can someone please explain the fix for IOS status bar
